### PR TITLE
docker-compose.yml variant to deploy HA

### PR DESCRIPTION
**Pull request by yamt**

### CODE REVIEW

**Comment by yamt** [comment]
https://github.com/rancher/k3s/pull/693 is necessary to make this work

File: docker-compose-ha.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    environment:
+    - K3S_STORAGE_BACKEND=etcd3
+    - K3S_STORAGE_ENDPOINT=etcd:2379
+    volumes:
+    - k3s-server2:/var/lib/rancher/k3s
+    ports:
+    - 16443:6443
+
+  node:
+    image: rancher/k3s:v0.8.0-rc1
```
Comment (erikwilson) [inline docker-compose-ha.yml]:
```suggestion
    image: rancher/k3s:v0.8.0
```

Comment (yamt) [reply docker-compose-ha.yml]: oops

**Comment by erikwilson** [comment]
Thanks!

**Pull request merged**